new-pipeline/src/dashboards.js

```diff
@@ -161,17 +154,22 @@ $(document)
             if (shortUrl.indexOf(":") === 4) {
               shortUrl = "https" + shortUrl.substring(4);
             }
-            $this.parents(".permalink-control")
-              .find("input")
-              .show()
-              .val(shortUrl)
-              .focus();
+            $this.parents(".navbar-form")
+            .find("input")
+            .show()
+            .val(shortUrl)
+            .focus();
           },
           async:false
         });
         document.execCommand('copy');
       });
-  });
```

Comment:
Does this code run? It looks as though you've dropped a closing sequence here.

Reply:
I did. I've added it back in with a note because I think that's what was confusing me. I kept the event handler within the ready doc function - I think that's appropriate? Could you take a look and let me know?

Thanks!